optaplanner-docs/src/main/asciidoc/BenchmarkingAndTweaking/BenchmarkingAndTweaking-chapter.adoc

```diff
@@ -195,7 +195,7 @@ There are several ways to serialize a solution:
 
 
 [[xStreamSolutionFileIO]]
-==== ``XStreamSolutionFileIO``: Serialize To And From An XML Format
+==== `XStreamSolutionFileIO: serialize to and from an XML format
```

Comment:
Suggestion for separate PR (so not blocking this PR):
Here ` are remove, but above there are added.